back/package.json

```diff
@@ -8,6 +8,9 @@
     "start:dev": "nodemon --exec babel-node --extensions \".ts\" src/index.ts",
     "start:debug": "run-p -l type-check:watch \"start:dev -- --inspect-brk\"",
     "start:prod": "node ./dist/index.js",
+    "start:mock-repository": "cross-env MOCK_RESPOSITORY=true npm run start",
```

Comment:
Typo

MOCK_REPOSITORY

Reply:
Typo fixed. Ready for review and merge if everything is ok now